base/ca/src/main/java/org/dogtagpki/server/ca/v2/EmptyACLFilter.java

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright Red Hat, Inc.
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
+//
+package org.dogtagpki.server.ca.v2;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebFilter;
+
+import org.dogtagpki.server.v2.ACLFilter;
+
+@WebFilter(servletNames = {"caInfo", "caCert", "caCertRequest"})
+public class EmptyACLFilter extends ACLFilter {
+
+    private static final long serialVersionUID = 1L;
+
+    @Override
+    public void init() throws ServletException {
+        setAcl("");
```

Comment:
Is this necessary? IIUC the `acl` is `null` by default so the filter will be bypassed.

Reply:
Currently, the filter has to be associated with the servlet also for empty acl  because it is responsible of the audit. If this acl is null the check will fail but I can change the check so we do not need to override the init for the empty  filter.